README.md

```diff
@@ -28,5 +28,12 @@ The following capability providers are included automatically in every host runt
 
 While neither of these providers requires a _link definition_, to use either of them your actors _must_ be signed with their contract IDs.
 
+## Community Capability Providers
+The following is a list of community supported capability providers developed by members of the wasmCloud community. Please see the [CONTRIBUTING](./CONTRIBUTING.md) guide for information on how to submit your capability provider.
+
+| Provider | Contract | Description |
+| :--- | :--- | :--- |
+| [kvdynamodb](https://github.com/matthewtgilbride/kvdynamodb-provider) | [`wasmcloud:keyvalue`](https://github.com/matthewtgilbride/kvdynamodb-provider) | AWS DynamoDB implementation for the keyvalue contract |
```

Comment:
I've just gotten around to giving some TLC to these repos (tests, READMEs).  I think implementing the rest of the `wasmcloud:keyvalue` contract is doable, but what about the _additional_ operation (`keys`) that drove me to create a separate [contract](https://github.com/matthewtgilbride/kvdynamodb)?  

Setting aside the larger questions of how to *version* interfaces and *extend* interfaces, do we think we can reach a consensus on whether `keys` should exist in `wasmcloud:keyvalue` and if so, the details w.r.t. paging?

Happy to take this conversation to a specific issue.

Reply:
Great question, I think this is something we should bring up in slack or in the community call, because it sounds like that was pretty core to your implementation

Reply:
We talked about it in the community call today, and the consensus was that the `keys` operation does not belong in `wasmcloud:keyvalue` for two main reasons
1. Some keyvalue stores don't implement key listing functionality (e.g. hashicorp vault, for security purposes)
2. Some keyvalue stores implement key listing functionality in a dangerous way (e.g. Redis single threaded `keys *`)

In order to avoid this ambiguity when selecting a contract, we'll leave it out for now. However, @matthewtgilbride I'd love to get a sense for your use case and figure out if we could tackle the need to list keys in a different way. For example, if we kept track of lists of objects in a keyvalue store, the `keys` requirement may be doable with just get + setadd operations.

We have a call tomorrow where we'll talk about this, so I can update this then as well 😄